portal-impl/src/com/liferay/portal/plugin/PluginPackageImpl.java

```diff
@@ -166,11 +164,7 @@ public String getVersion() {
 
 	@Override
 	public int hashCode() {
-		HashCodeBuilder hashCodeBuilder = new HashCodeBuilder();
-
-		hashCodeBuilder.append(getModuleId());
-
-		return hashCodeBuilder.hashCode();
+		return getModuleId().hashCode();
```

Comment:
@julschong Method chain here is not allowed, please do:
String moduleId = getModuleId();
return moduleId.hashCode();